packages/adminrouter/docker/Dockerfile

```diff
@@ -120,7 +120,7 @@ COPY adminrouter-redirect-http-https.conf \
      adminrouter-listen-agent.conf \
      adminrouter-tls.conf \
         /opt/mesosphere/etc/
-COPY ca.crt /run/dcos/pki/CA/certs/
+COPY ca.crt /run/dcos/pki/CA/certs/ca-bundle.crt
```

Comment:
This will break AR:

https://github.com/mesosphere/dcos-enterprise/blob/master/packages/adminrouter/extra/src/nginx.master.conf#L62